src/internal/packager/helm/zarf.go

```diff
@@ -49,32 +49,19 @@ func (h *Helm) UpdateZarfRegistryValues() error {
 }
 
 // UpdateZarfGiteaValues updates the Zarf git server deployment with the new state values
-func (h *Helm) UpdateZarfGiteaValues() error {
-	giteaValues := map[string]interface{}{
-		"gitea": map[string]interface{}{
-			"admin": map[string]interface{}{
-				"username": h.cfg.State.GitServer.PushUsername,
-				"password": h.cfg.State.GitServer.PushPassword,
-			},
-		},
-	}
-
-	h.chart = types.ZarfChart{
-		Namespace:   "zarf",
-		ReleaseName: "zarf-gitea",
-	}
+func (h *Helm) UpdateZarfGiteaValues(oldState *types.ZarfState) error {
```

Comment:
This should probably be moved from Helm because it isn't really interacting with Helm anymore and should be in gitea.go instead.

Reply:
Still new to Go so i'm not sure if there's a better way, but when I moved this method out of helm and into gitea.go i also had to move the new git client instantiation out of the method and into the update-creds cmd code in order to call the method.